{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "1394062c_45c580cb",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1372423
      },
      "writtenOn": "2024-05-27T13:44:47Z",
      "side": 1,
      "message": "WDYT?",
      "revId": "d55cba4c972674ba884fc1f28037ae2705c6b611",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "9108bda1_e57e609f",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 1524702
      },
      "writtenOn": "2024-05-28T09:27:34Z",
      "side": 1,
      "message": "This doesn\u0027t fix the test failure related to `assertThrottled`, right?",
      "revId": "d55cba4c972674ba884fc1f28037ae2705c6b611",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba"
    }
  ]
}